src/index.ts

```diff
@@ -1,13 +1,14 @@
 /* istanbul ignore file */
-import * as conversionUtils from './utils/convert';
-// import { FeatureRegistryTransactionParams } from './contract_wrappers/registries/feature_registry_wrapper';
+import { FeatureRegistryTransactionParams } from './contract_wrappers/registries/feature_registry_wrapper';
+import { GeneralTransferManagerTransactionParams } from './contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper/common';
+import {SecurityTokenTransactionParams} from './contract_wrappers/tokens/security_token_wrapper';
```

Comment:
Separate brackets from variable name (also should be commented)

Reply:
Jere, why do you have it commented?

I have them uncommented because I am using some of the transactionParams in my examples, for example using the general transfer manager to get the kyc data params structure. I can remove the interfaces from the code but let me know

Reply:
I commented them because I thought we weren't gonna use them until we figure out how to make them compatible with the multi version structure we use now.

If you're using them in the examples I guess it's fine to leave them uncommented, so that when we do apply the change we'll know to adapt the examples because the editor will start complaining